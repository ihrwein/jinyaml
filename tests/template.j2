{
	"reader_user_DN" : "{{reader_user_DN}}",
	"reader_user_password" : "{{reader_user_password}}",
	"search_base" : "{{search_base}}",
	"module_DN_template" : "{{module_DN_template}}",
	"url" : "{{url}}",
	"scope" : {{scope}}
}
