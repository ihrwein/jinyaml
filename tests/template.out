{
	"reader_user_DN" : "username",
	"reader_user_password" : "password",
	"search_base" : "OU=AdminView,dc=example,dc=org",
	"module_DN_template" : "CN={},OU=modules,OU=AdminView,dc=example,dc=org",
	"url" : "ldaps://example.org",
	"scope" : 2
}
